build/container/external/packages/aio/00-makefile.patch

```diff
@@ -0,0 +1,43 @@
+--- src/Makefile	2016-11-30 09:59:12.889526296 -0800
```

Comment:
yep. welcome to package patches. when we update the package there is a chance we need to regenerate patches.